crates/e2e/tests/e2e/cow_amm.rs

```diff
@@ -290,7 +284,7 @@ async fn cow_amm_jit(web3: Web3) {
     // place user order with the same limit price as the CoW AMM order
     let user_order = OrderCreation {
         sell_token: *onchain.contracts().weth.address(),
-        sell_amount: alloy::primitives::U256::from(10).pow(alloy::primitives::U256::from(17)), // 0.1 WETH
+        sell_amount: U256::from(10).pow(U256::from(17)), // 0.1 WETH
```

Comment:
Would be nice to extend the `EthUnit` trait to work with `f64` to make this a `0.1.eth()`.

Reply:
> the other units break consistency because the parse_unit functions takes eth specific ones (wei, etc)

This will only come up inside the implementation of `EthUnit`, though. IMO from the "outside" it doesn't matter if the implementation thinks it's working with ETH when we use it for USDC as long as it spits out the expected number.